cluster/sbin/snappy-nodes.sh

```diff
@@ -338,12 +338,41 @@ function getNumLeadsOnHost() {
   echo $numLeadsOnHost
 }
 
+function scpConf() {
+ var=$(ip addr | grep 'state UP' -A2 | tail -n1 | awk '{print $2}' | cut -f1  -d'/')
+ if [[ "$host" != "$var" && "$host" != "localhost" ]] ; then
```

Comment:
The user can give the actual hostname instead of 'localhost' or 'ip' then in that case this condition will be incorrect.

Reply:
Modified as per review